doc/sphinx/conf.py

```diff
@@ -140,7 +140,14 @@ def setup(app):
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
-#html_use_smartypants = True
+# Better to disable this to prevent alteration of command-line arguments
+# Sphinx <1.6 use `html_use_smartypants` and >=1.6 use `smartquotes`
+from sphinx import __version__ as sphinx_version
+sphinx_version_parts = [int(i) for i in sphinx_version.split('.')]
+if sphinx_version_parts[0] <= 1 and sphinx_version_parts[1] < 6:
+    html_use_smartypants = False
+else:
+    smartquotes = False
```

Comment:
It would be nice if CI was generating the documentation too so that we could see the result before merging. But also easier to adapt in case of issues.

Reply:
True that!! I'll see what I can do.